### DISCUSSION TITLE
How do I add translations without editing the library?

### DISCUSSION THREAD

**Discussion by dawidprosba** [react-page/react-page#863] [Q&A] [answered]
Is there a way to translate the editor in simillar way as plugins can be translated?

I figured out that some of the editor's components, eg. `Inner` have  `defaultTranslations` variables.
But I dont really see in what way the `translations` props can be passed to that component from the <Editor /> (@react-page/editor) component or how to override them without modifying the library.

**Comment by macrozone** [comment, marked as answer]
i am focusing right now on huge refactorings and will tackle translations at some point after the refactoring

Its best to open up a issue on what you d' like to have translated. Would also love to see suggestions on how to define translations.
